xbmc/addons/VFSEntry.cpp

```diff
@@ -134,15 +134,30 @@ class CVFSURLWrapper
     std::vector<std::string> m_strings;
 };
 
+CVFSEntry::ProtocolInfo::ProtocolInfo(BinaryAddonBasePtr addonInfo)
+  : supportPath(addonInfo->Type(ADDON_VFS)->GetValue("@pathSupport").asBoolean()),
+    supportUsername(addonInfo->Type(ADDON_VFS)->GetValue("@usernameSupport").asBoolean()),
+    supportPassword(addonInfo->Type(ADDON_VFS)->GetValue("@passwordSupport").asBoolean()),
+    supportPort(addonInfo->Type(ADDON_VFS)->GetValue("@portSupport").asBoolean()),
+    supportBrowsing(addonInfo->Type(ADDON_VFS)->GetValue("@browsingSupport").asBoolean()),
```

Comment:
Should we use the same wording like in the struct? `supportPath` etc

Reply:
fine.